public/sitemap.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove this file as well.

A main idea of an initial task was about providing jobs meta-data to Open Graph snippets, not about sitemaps.

Reply:
I've added them because of this comment on job messages: "If you will find any useful improvements to make OptriSpace more SEO-friendly – please let me know :-)"
I'm removing it in next commit.

Reply:
Of course we can discuss your interesting ideas, proposals and future changes, but do not to implement in this task ASAP (and in the same PR) 😁

This specific task was just about meta-tags to make snippets more SEO-friendly.

Reply:
Sorry, you are right. I'm removing all sitemap things from code.